src/components/ui/remember-checkbox/remember-checkbox.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import { UseFormReturn } from 'react-hook-form';
+
+type Props = {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  form: UseFormReturn<any, any, any>;
+};
+
+export const RememberCheckbox = ({ form }: Props) => {
+  const [checked, setChecked] = useState(false);
```

Comment:
if u are using react hook form, u don't need to keep this state here, u can store in form and get it from form directly

Reply:
I dont know, for some reasons it didnt work without state, but now it works